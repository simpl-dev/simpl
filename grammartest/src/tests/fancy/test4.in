foo2 a b