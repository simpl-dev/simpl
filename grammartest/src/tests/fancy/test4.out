0:7:Program {
  foo=(null)
  foo2=0:7:Foo2 {
    bar=7:7:Id {
      text="b"
    }
    foo=5:5:Id {
      text="a"
    }
  }
}
